modules/kafka-cluster/inputs.tf

```diff
@@ -30,5 +23,20 @@ variable "region" {
   description = "The cloud service provider region where the Kafka cluster is running - see supported list - https://docs.confluent.io/cloud/current/clusters/regions.html#cloud-providers-and-regions"
   type        = string
   default     = "asia-southeast2"
+}
 
+variable "cluster_type" {
+  description = "The value can be either basic, standard, and dedicated."
+  type        = string
+  default     = "basic"
```

Comment:
Minor: We could add a validation of the input, to verify is either "basic" or "standard" or "dedicated"

Reply:
yes good suggestion i have updated the same